_posts/2021-02-12-generalist-specialization-and-the-impostor-syndrome.md

```diff
@@ -0,0 +1,80 @@
+
+Also, note that typically this moment of self-doubt also means that at your worst moment you're trying to compare yourself to other people at their best moment. Or even worse, trying to compare yourself to an ideal image of yourself. It's an unfair advantage, and one you will always lose. There is always someone who knows more about something.
+
+According to some studies made on the subject, the main medication is just to try to reframe/rephrase that self-doubt with a bit more bright way of looking at things. For example, if your thought is: "I'm out of my league, I don't know how to do this!" - then you could try to reframe it like "This seems difficult, but I have time to educate myself, and everybody understands I'm not a master of everything. I am allowed to fail."  
+
+This actually goes nicely with agile methodologies, too, because a failure IS an option - you just try to fail fast in that case, take your learnings, and build on top of that. Transparency also helps - the feeling of being a fraud often comes if you hide your internal worries deep, and try to set up a facade. As with any self-doubt, it might do marvels if you just have a chat with your peers, or project lead, or life coach, and are honest about your worries.  
+
+So if you feel unsure, stop for a bit, take a deep breath, and try to see things from another angle. Try this question:
+
+**"What would I need to change to be able to succeed?"**
+
+## Impostor syndrome as a positive thing?
+
+The analytical part of my brain understands all this clearly. But unfortunately, feelings are not something that can be dismissed by logical thought and willpower alone. One way of course to avoid those feelings would be to stop pushing yourself into those situations anymore. If you know one thing well, stick to that one thing? Well, while you certainly could do that, I would not recommend it.
+
+See, growth comes from the combination of applying what you know but being also exposed to things you don't know - yet. At least for me, that might raise the appetite to know more, to understand more, to learn more. And that is half of why I love the software developer life (The other half is instant gratification from solving concrete problems). As I recognize that feeling for what it is, I can avoid letting it overwhelm me or bring stress on me.
+
+So for me, the impostor syndrome visiting is actually a signal that I've once again managed to push myself to a new uncomfortable zone, where I don't already know everything. So I've managed to create a learning and growth opportunity for this year, too. And that's exactly why I mean it when I say:
```

Comment:
"the impostor syndrome visiting" -> "a visit from impostor syndrome"

"zone, where" -> "zone where"